ably/rest/rest.py

```diff
@@ -65,6 +65,10 @@ def __init__(self, key=None, token=None, token_details=None, **kwargs):
         # else:
         #     self.__session = None
 
+        self.__api_version = "0.9"
```

Comment:
It seems we now duplicate version numbers, see https://github.com/ably/ably-python/blob/master/setup.py#L8.  We need to ensure that the library only has version numbers stored in one place.